python/pyspark/sql/session.py

```diff
@@ -432,14 +430,9 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
             ``byte`` instead of ``tinyint`` for :class:`pyspark.sql.types.ByteType`. We can also use
             ``int`` as a short name for ``IntegerType``.
         :param samplingRatio: the sample ratio of rows used for inferring
+        :param verifySchema: verify data types of every row against schema.
         :return: :class:`DataFrame`
 
-        .. versionchanged:: 2.0
```

Comment:
Out of interest why are we removing this note but keeping the other 2.0 change note? Just wondering so that when I'm making my changes for 2.1 I can do the right thing.